backend/items/management/commands/export_to_ndjson.py

```
@@ -0,0 +1,168 @@
+        if output_file.exists() and not overwrite:
+            self.stdout.write(self.style.WARNING(f"Skipping all_data.ndjson (already exists, use --overwrite)"))
+            return
+        
+        if verbose:
+            self.stdout.write("Creating combined all_data.ndjson...")
+        
+        with open(output_file, 'w', encoding='utf-8') as out_file:
+            for ndjson_file in ['users.ndjson', 'items.ndjson', 'locations.ndjson']:
+                file_path = output_dir / ndjson_file
+                if file_path.exists():
+                    with open(file_path, 'r', encoding='utf-8') as in_file:
```

Ruff (0.15.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)